### ISSUE TITLE
"terraform fmt" continually adds spaces to multiline comment blocks

### ISSUE THREAD

**Issue by cnuss** [hashicorp/terraform#16639]
### Terraform Version
v0.10.8

### Terraform Configuration Files
```hcl

resource "aws_instance" "instance" {
  /* my
  multiline
  comment
  */
  ami = "ami-abcdef1234"
}
```

(Notice there's NO newline between the end of the comment block and `ami = ...`)

### Expected Behavior
`terraform fmt` should properly format comment blocks.  Subsequent calls to `terraform fmt` on an unmodified file should add spaces to comment blocks.  Newlines between comment blocks and properites 

### Actual Behavior
Adding a comment block with no newline between the comment and a property causes `terraform fmt` to continually add spaces

### Steps to Reproduce
Please list the full steps required to reproduce the issue, for example:
1. create a HCL file similar to above
1. run `terraform fmt` (spaces are added in the comment block)
1. run `terraform fmt` again (spaces are added AGAIN in the comment block)
1. add a newline between `/*` and `ami = ...`
1. run `terraform fmt`multiple times, it's formatted properly, subsequent `fmt` calls don't add spaces.

### Important Factoids
Adding a newline between the end of the comment and the next properly stops `fmt` from adding additional spaces

Additional spaces are added:
```
resource "aws_instance" "instance" {
  /* my
  multiline
  comment
  */
  ami = "ami-abcdef1234"
}
```

Additional spaces are NOT added:
```
resource "aws_instance" "instance" {
  /* my
  multiline
  comment
  */
  
  ami = "ami-abcdef1234"
}
```

**Comment by jason-azze** [comment]
I can confirm this is still true in 0.11.1. And now I don't have to open a new issue. :smiley:

**Comment by nodakai** [comment]
Reproducible with 0.11.7

**Comment by apparentlymart** [comment]
Hi all!

The `terraform fmt` command has been reworked as part of Terraform v0.12 and that fix is now in the master branch, ready to be included in the forthcoming v0.12.0 release. Since it's now merged, I'm going to close this out. Thanks for reporting this, and thanks for your patience while we got it done.
